pkg/agent/waitfor.go

```diff
@@ -26,13 +26,22 @@ func WaitForBootstrapComplete(assetDir string) (*Cluster, error) {
 	waitContext, cancel := context.WithTimeout(cluster.Ctx, timeout)
 	defer cancel()
 
+	var lastErr error
 	wait.Until(func() {
-		bootstrap, err := cluster.IsBootstrapComplete()
+		bootstrap, err, exit_on_err := cluster.IsBootstrapComplete()
 		if bootstrap && err == nil {
 			logrus.Info("cluster bootstrap is complete")
 			cancel()
 		}
 
+		if err != nil {
+			logrus.Info(err)
```

Comment:
I think we should only have this Info log if exit_on_err is false.

Reply:
Changed to log here if not exiting.